docs/cluster/integrations/index.md

```
@@ -54,3 +64,18 @@ specific data source. This allows secure access to the external system and is
+
+:::
+### Integration types
+:::
+
+There are 3 different integration types:
+- **Full load only**: Imports all your data and immediately ends after.
+- **CDC only**: It indefinitely listens to CDC (Change Data Capture) events on the 
+  source and applies them into your CrateDB table. Once it reaches the last CDC event 
+  it waits for new events to come.
+- **Full load and CDC**
+  It imports all the data like the type __full load only__, but once that phase finishes
```

markdownlint-cli2 (0.21.0)

[warning] Strong style
Expected: asterisk; Actual: underscore

(MD050, strong-style)